cirq-core/cirq/linalg/transformations.py

```diff
@@ -807,6 +806,15 @@ def transpose_flattened_array(t: np.ndarray, shape: Sequence[int], axes: Sequenc
     return ret
 
 
+@functools.cache
+def can_numpy_support_dims(num_dims: int) -> bool:
```

Comment:
If that function is intended for reuse elsewhere, please add docstring.
Otherwise rename to `_can_numpy...`.

Reply:
done